lib/ohai/plugins/packages.rb

```diff
@@ -31,6 +31,7 @@
   end
 
   collect_data(:linux) do
+    require "date"
     packages Mash.new
     if %w{debian}.include? platform_family
```

Comment:
Any chance you can change these platform_family checks into a case statement. At 3 different checks of the same data it would be faster to use a single check in a case statement